modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -9578,6 +9578,7 @@ logger-name=Logger Name
 logging-audit-message-processor-configuration-name=Logging Message Audit Message Processor
 login=Login
 login-configuration-name=Login
+login-create-account-allow-custom-password=Allow Custom Password At Account Creation
```

Comment:
@thabet91 Just FYI, when we're making changes to match QA's verification result, we'll need to adjust the text here as well. As BChan's recommendation using https://capitalizemytitle.com, we'll put this text as `Allow Custom Password at Account Creation` instead.